src/main/java/model/UserModel.java

```diff
@@ -0,0 +1,31 @@
+package model;
+
+import beans.UserBean;
+import dao.DaoFactory;
+import helper.MailSender;
+import helper.Password;
+
+public class UserModel {
+	
+	private DaoFactory daoFactory;
+	
+	public UserModel() {
+		this.daoFactory = new DaoFactory();
+	}
+	
+	public boolean registerUser(UserBean user) {
+		boolean result = false;
+		
+		if(user.getEmail() != null) {
```

Comment:
getEmail() will never return a null value so this if completely useless, if you wish to check for something you can check for empty values or check if the email is of valid format